test-e2e/app/ambassador/ambassadors.page.e2e.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * Created by vs on 4/19/16.
+ */
+
+describe('Ambassador PAge', function () {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 100000;
+
+
+    var lastOgranization = element(by.css('.ambassadors-organization.show>img'));
+    var headerH2 = element(by.css('h2[class="heading"]'));
+    var logoHeader = element(by.css('.logo.pull-left>img'));
+    var menuHeader = element(by.css('.menu-icon'));
+    var contentH2 = element.all(by.css('div[class="ambassadors-peoples"] > h2'));
+    var ambassadorsPeopleIMG = element.all(by.css('.ambassadors-people>img'));
+    var ambassadorsPeopleNames = element.all(by.css('.name'));
+    var ambassadorsPeopleCountries = element.all(by.css('.country'));
+    var ambassadorsOrganizationsIMG = element.all(by.css('.ambassadors-organization.show>img'));
+    var seeMore = element.all(by.css('.custom-button'));
+
+    var EC = protractor.ExpectedConditions;
+    var TIMEOUT = 10000;
+    browser.manage().timeouts().pageLoadTimeout(75000);
+
+    beforeAll(function () {
+        browser.get('/ambassadors');
```

Comment:
The beforeAll function is called only once before all the specs in describe are run, and the afterAll function is called after all specs finish.

http://jasmine.github.io/2.1/introduction.html